src/marqo/s2_inference/hf_utils.py

```diff
@@ -16,15 +24,95 @@ def __init__(self, *args, **kwargs) -> None:
 
         if self.max_seq_length is None:
             self.max_seq_length = 128
+        self.model_properties = kwargs.get("model_properties", dict())
+        self.model_name = self.model_properties.get("name", None)
+
+        model_auth = kwargs.get(InferenceParams.model_auth, None)
```

Comment:
Can't this just be `self.model_auth = kwargs.get(InferenceParams.model_auth, None)`